test/rules/assertions/requireJsdoc.js

```diff
@@ -3180,6 +3180,91 @@ function quux (foo) {
       }
       `,
     },
+    {
+      code: `
+      function comment () {
+        return "comment";
+      }
+      `,
+      errors: [
+        {
+          line: 2,
+          message: 'Missing JSDoc comment.',
+        },
+      ],
+      ignoreReadme: true,
```

Comment:
Notwithstanding the fact that we do need more `ignoreReadme` usage for some obscure edge cases, I think we actually want these examples to not be ignored as they demonstrate an option.

```suggestion
```